packages/grafana-llm-app/pkg/plugin/anthropic_provider.go

```diff
@@ -56,6 +56,20 @@ func (p *anthropicProvider) Models(ctx context.Context) (ModelResponse, error) {
 	}, nil
 }
 
+func (p *anthropicProvider) forceUserMessage(req *openai.ChatCompletionRequest) {
+	hasUserMessage := false
+	for _, message := range req.Messages {
+		if message.Role == "user" {
+			hasUserMessage = true
+			break
+		}
+	}
+
+	if !hasUserMessage {
+		req.Messages[len(req.Messages)-1].Role = "user"
```

Comment:
Could we short circuit (and possibly return error?) if there are no messages?

Reply:
Going to just return if there are no messages.

Throwing error on empty messages should probably be handled else where in the plugin.

We currently rely on features themselves to handle errors coming from the APIs and when they don't handle it, the frontend just hangs.

Reply:
Just returning sounds good to me, wouldn't change current behavior and avoids the panic/DOS issue.